app/views/articles/create.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.id @article.id
```

Comment:
you can tell the `create` action/method in the controller to use the `show` jbuilder, by using `render :show`. That way you don't have two different templates that do the same thing